### ISSUE TITLE
Table header appears over the Side Panel

### ISSUE THREAD

**Issue by ShehanIshanka** [h2oai/wave#2020]
Wave SDK: 0.25.3 , OS: MacOs

#### Actual behavior

When the Table is scrolled through and opens the side panel, the header bar will appear over the side panel.

https://github.com/h2oai/wave/assets/29954570/efbc52cc-b889-4990-81c2-6d9382e37013

```python
from h2o_wave import main, app, Q, ui

@app("/")
async def serve(q: Q):
    if not q.client.initialized:
        q.page["meta"] = ui.meta_card(box="")
        q.page["form"] = ui.form_card(
            box="1 1 -1 6",
            items=[
                ui.table(
                    name="issues",
                    columns=[
                        ui.table_column(
                            name="text1", label="Issues reported by", searchable=True
                        ),
                        ui.table_column(name="text2", label="Issue"),
                    ],
                    rows=[
                        ui.table_row(
                            name=f"row{i}", cells=[f"Issue{i}", f"Issue is {i}"]
                        )
                        for i in range(100)
                    ],
                    height="500px",
                )
            ],
        )
        q.client.initialized = True
    else:
        if q.args.issues:
            q.page["meta"].side_panel = ui.side_panel(
                title="Welcome to store",
                items=[
                    ui.text("Fake text"),
                ],
                closable=True,
                events=["dismissed"],
            )

    if q.events.my_side_panel:
        if q.events.my_side_panel.dismissed:
            q.page["meta"].side_panel = None

    await q.page.save()
```

#### Expected behavior

Header bar should not be appeared over side panel.

**Comment by ShehanIshanka** [comment]
This is a blocker for the new MLOps release since we used few changes in `0.25.3` to it. In that case, can we get a patch fix?
cc: @shereezhang @dulajra @shihan007

**Comment by mturoci** [comment]
@marek-mihok this should be already fixed via #1980. Can you please verify?

**Comment by marek-mihok** [comment]
> @marek-mihok this should be already fixed via https://github.com/h2oai/wave/issues/1980. Can you please verify?

@mturoci yes, it's fixed. @ShehanIshanka the fix will be a part of the next release this or next week.

https://github.com/h2oai/wave/assets/23740173/99488797-c1ba-4e03-aac4-77bc04ecaf65
